apps/xui/xui-webapp-multiday-hearings-integration/image-policy.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: xui-webapp-multi-day-hearings-integration
```

Fix more such references. ```suggestion
  name: xui-webapp-multiday-hearings-integration
```

---

updated thx @adusumillipraveen